Documentation/data-cleanup/python-script/resource_delete.py

```
@@ -0,0 +1,246 @@
+
+            collection_id = mb.get_collection_id(solution_id)
+            mb.delete_collection(collection_id)
+
+            groups = mb.get_permission_groups()
+            gid = mb.get_permission_group_id(groups, solution_id)
+            mb.delete_permission_group(gid)
+
+        else:
+            logger.info("Unknown event type; skipping.")
+
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)